cluster/src/main/scala/com/linkedin/norbert/cluster/Node.scala

```diff
@@ -77,8 +81,9 @@ object Node {
  * @param available whether or not the node is currently able to process requests
  * @param partitions the partitions for which the node can handle requests
  * @param capability the 64 bits Long representing up to 64 node capabilities
+ * @param offset the offset in the time cycle at which this node GCs
```

Comment:
This needs better documentation. An example might be useful here. We should also state that this is only used if we are doing GC aware load balancing.